packages/paste-website/src/pages/components/flex/index.mdx

```diff
@@ -123,7 +123,7 @@ you use the Flex as a custom element, it is up to you to manage the resulting ac
 
 <LivePreview scope={{Box, Flex}} language="jsx">
   {`<Flex>
-    <Flex grow shrink basis>
+    <Flex grow shrink basis={0}>
```

Comment:
question: Did we consider setting this to a non-zero value? Previously, as the issue points out, the prop value for `basis` would have been `true`. Is there a benefit to using a truthy value here?

Reply:
question: Why is the `basis` prop on line 135 unchanged? Could we please check the docs page once more (and prob the stories) to ensure we catch other invalid prop values?

Reply:
update from convo: I'll update all the `basis` props on this page to be valid values and I updated the AC of [this ticket](https://issues.corp.twilio.com/browse/DSYS-2903) to include adding more helpful info for the basis, grow, and shrink properties to the docs.